tests/16-cidr-ingress-policy.sh

```diff
@@ -121,7 +120,7 @@ docker run --rm -i --net ${TEST_NET} -l "${ID_SERVICE2}" --cap-add NET_ADMIN ${D
 set -e
 
 log "importing L3 CIDR policy for IPv4 egress"
-policy_delete_and_wait "--all"
+cilium policy delete --all
```

Comment:
Why were calls to `policy_delete_and_wait "--all"` removed?

Reply:
Functionally it is not necessary to wait after deletion as a new policy is being installed right after and we wait for that. This allows cilium to skip regenerating bpf for the deleted policy in some cases, depending on how fast the bpf rebuilds are scheduled after policy deletion.

Reply:
Makes sense, thanks.